src/collapse/perf.rs

```diff
@@ -815,6 +815,37 @@ mod tests {
         Ok(())
     }
 
+    #[test]
+    fn test_multiple_skip_after() -> io::Result<()> {
+        let path = "./tests/data/collapse-perf/go-stacks.txt";
+        let mut file = fs::File::open(path)?;
+        let mut bytes = Vec::new();
+        file.read_to_end(&mut bytes)?;
+        let mut folder = {
+            let options = Options {
+                skip_after: vec!["main.init".to_string(), "regexp.compile".to_string()],
+                ..Default::default()
+            };
+            Folder::from(options)
+        };
+        let mut buf_actual = Vec::new();
+        <Folder as Collapse>::collapse(&mut folder, &bytes[..], &mut buf_actual)?;
+        let lines = std::str::from_utf8(&buf_actual[..]).unwrap().lines();
+
+        for line in lines {
+            // without `skip_after` some collapsed lines would look like:
+            // go;[unknown];x_cgo_notify_runtime_init_done;runtime.main;main.init;...
+            if line.contains("main.init") {
+                assert!(line.contains("main.init;")); // we removed the frames above "main.init"
```

Comment:
Actually, reading this test again. I think this should be `line.starts_with`, not `line.contains`. Otherwise it doesn't seem to test much?

Reply:
Good point. Fixed them both, and confirmed that the tests fail if I remove things from `skip_after`.